angr/analyses/reaching_definitions/def_use_graph.py

```diff
@@ -0,0 +1,53 @@
+import networkx
+
+from .definition import Definition
+
+
+def _is_definition(node):
+    return isinstance(node, Definition)
+
+
+class DefUseGraph():
```

Comment:
Parentheses are redundant here.

Reply:
This is on top of #1869 , will be corrected when rebasing.